CONTRIBUTING.md

```diff
@@ -83,4 +83,13 @@ Changes to `buzzfeed/sso` happen via GitHub pull requests, which is where at lea
 
 [begin-guide]: https://akrabat.com/the-beginners-guide-to-contributing-to-a-github-project/
 [mindful-comms]: https://kickstarter.engineering/a-guide-to-mindful-communication-in-code-reviews-48aab5282e5e
-[go]: https://golang.org/doc/install
\ No newline at end of file
+[go]: https://golang.org/doc/install
+
+### Merge Policy
+
+In order to allow SSO to be the most useful, while balancing feature needs with security concerns, we take what we're calling a "generous merge/deprecate often" approach, which means:
```

Comment:
Can we call this "optimistic merging" and link to the article on it (http://hintjens.com/blog:106)

Reply:
Absolutely! Would you like the link inline or next to it? :)

Reply:
Inline works!